backend/src/common/packages/controller/controller-base.package.ts

```diff
@@ -64,7 +64,7 @@ class ControllerBase implements Controller {
             };
         }
         return {
-            body: { ...(body as object), file },
+            body: { ...(body as object) },
```

Comment:
Can we leave just like this ? :
```suggestion
            body,
```